cpu/efm32/include/periph_cpu.h

```diff
@@ -357,28 +357,23 @@ typedef struct {
     timer_dev_t prescaler;  /**< the lower neighboring timer (not initialized for LETIMER) */
     timer_dev_t timer;      /**< the higher numbered timer */
     IRQn_Type irq;          /**< number of the higher timer IRQ channel */
+    uint8_t ch_numof;       /**< number of channels per timer */
 } timer_conf_t;
 /** @} */
 
 
 /**
- * @brief   The implementation can use one LETIMER or two regular timers cascaded
+ * @brief   Use LETIMER as the base timer for XTIMER
  */
-#ifndef CONFIG_EFM32_USE_LETIMER
-#define CONFIG_EFM32_USE_LETIMER   0
+#ifndef CONFIG_EFM32_XTIMER_USE_LETIMER
+#define CONFIG_EFM32_XTIMER_USE_LETIMER   0
 #endif
 
-#if IS_ACTIVE(CONFIG_EFM32_USE_LETIMER)
 /**
- * @brief   This timer implementation has two available channels
+ * @brief   This timer implementation has two or three available channels
+ *          depending on the used timer, use the lowest.
  */
 #define TIMER_CHANNEL_NUMOF     (2)
```

Comment:
Where is this macro used now?

Reply:
Only needed because `tests/periph_timer_periodic/main.c` relies on it..., I can probably adapt the test as well though..

Reply:
Removed.